bot.py

```diff
@@ -79,14 +79,16 @@ class DeliveryType(enum.Enum):
     },
     'dragon_face': {
         'desc': 'Wok',
-        'instr': "Check the menu at: https://www.thuisbezorgd.nl/iwok-go\n"
+        'instr': "Check the menu at: https://nijmegen.iwokandgo.nl/\n"
                  "Don't forget to ask for chopsticks!",
+                 "Destination: 6525EC Toernooiveld 212, ~18:00",
```

Comment:
I don't think the time matters, it's better to tell the person when to order (17:30). Please also add the delivery address to the Pizza option